src/core/server/core_usage_data/core_usage_data_service.ts

```diff
@@ -138,10 +154,42 @@ export class CoreUsageDataService implements CoreService<CoreUsageDataSetup, Cor
           });
       })
     );
+  }
 
-    return {
-      indices,
-    };
+  private async getSavedObjectAliasUsageData(elasticsearch: ElasticsearchServiceStart) {
+    const { body: resp } = await elasticsearch.client.asInternalUser.search({
+      index: this.kibanaConfig!.index,
+      body: {
+        track_total_hits: true,
+        query: { match: { type: LEGACY_URL_ALIAS_TYPE } },
+        aggs: {
+          aliases: {
```

Comment:
I modeled this after the Spaces usage data collection, was not aware `SO.find` supports `aggs` 😄 

As discussed offline: `aggs` doesn't support `filters`, yet, so we can't change it. But I'll add a comment here noting that it can be changed in the future.